lightning/src/ln/channel.rs

```diff
@@ -1447,27 +1447,27 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 	fn funding_created_signature<L: Deref>(&mut self, sig: &Signature, logger: &L) -> Result<(Transaction, LocalCommitmentTransaction, Signature), ChannelError> where L::Target: Logger {
 		let funding_script = self.get_funding_redeemscript();
 
-		let local_keys = self.build_local_transaction_keys(self.cur_local_commitment_transaction_number)?;
-		let local_initial_commitment_tx = self.build_commitment_transaction(self.cur_local_commitment_transaction_number, &local_keys, true, false, self.feerate_per_kw, logger).0;
-		let local_sighash = hash_to_message!(&bip143::SighashComponents::new(&local_initial_commitment_tx).sighash_all(&local_initial_commitment_tx.input[0], &funding_script, self.channel_value_satoshis)[..]);
+		let keys = self.build_local_transaction_keys(self.cur_commtment_transaction_number)?;
+		let initial_commitment_tx = self.build_commitment_transaction(self.cur_commtment_transaction_number, &keys, true, false, self.feerate_per_kw, logger).0;
```

Comment:
I interpret code here as "We build a commitment transaction. Owner is us" So no prefix.

Reply:
Hmm, so I thought this would have fallen under the second bullet in the linked comment. Would we ever say `local_commitment_tx` and `remote_commitment_tx` in your interpretation? If so, in what context?

Reply:
I think we should restrain pointed distinction (local/remote) to templates as used by our chan helpers (`chan_utils.rs`) and avoid leaking local/remote in Channel storage fields. Even it's not perfect for now.